# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/09
# do not build  mpi4py, and nfft as loaded as dependencies
diff -ru PyTom-1.0/pytom/pytomc/compile.py PyTom-1.0_disable_external_build/pytom/pytomc/compile.py
--- PyTom-1.0/pytom/pytomc/compile.py	2022-10-13 15:58:17.000000000 +0200
+++ PyTom-1.0_disable_external_build/pytom/pytomc/compile.py	2022-10-13 17:28:03.818142209 +0200
@@ -273,9 +273,9 @@
 print(setflags_line)
 print('')
 
-nosh       = False # you can choose not to compile the SH Alignment library
-nompi4py   = False  # you can choose not to compile the mpi4py library
-nonfft     = False  # you can choose not to compile the NFFT library
+nosh       = False # you can choose not to compile the SH Alignment library
+nompi4py   = True  # you can choose not to compile the mpi4py library
+nonfft     = True  # you can choose not to compile the NFFT library
 novoltools = True
 
 
